src/common/transformations/src/transformations/common_optimizations/mark_subgraphs_to_keep_in_mixed_precision.cpp

```diff
@@ -0,0 +1,173 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/mark_subgraphs_to_keep_in_mixed_precision.hpp"
+
+#include <ngraph/op/util/broadcast_base.hpp>
+#include <ngraph/op/util/gather_base.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <openvino/pass/pattern/op/or.hpp>
+
+#include "itt.hpp"
+#include "ngraph/env_util.hpp"
+#include "openvino/opsets/opset1.hpp"
+#include "openvino/opsets/opset3.hpp"
+#include "openvino/opsets/opset8.hpp"
```

Comment:
Can it be better to list individual operations instead of adding 3 relatively large headers?

Reply:
Lets not change it for now.

Reply:
For a couple of operations in the pattern we need to specify different versions: v3 and newer ones. 

But `#include opset1.hpp` agree is redundant

Reply:
Removed `opset1`, renamed `opset8 -> opset10`. Only `opset1` and `opset10` are left.